runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileContent/ProcessTileContent.styles.ts

```diff
@@ -1,14 +1,34 @@
-import { CardContent } from '@mui/material';
+import { CardContent, Container, Box } from '@mui/material';
 import styled from 'styled-components';
 
 export const StyledContent = styled(CardContent)`
     display: grid;
-    grid-template-columns: 1fr 1.5fr;
-    grid-template-rows: repeat(2, auto);
-    justify-items: start;
-    justify-content: space-evenly;
+    grid-template-columns: minmax(0, 1fr) auto minmax(0, 1fr);
+    column-gap: 5px;
+    height: 100%;
     width: 100%;
 
-    padding: 0 1.5rem 1rem 1.5rem;
-    gap: 1rem;
+    && {
+        padding: 15px 10px;
+    }
 `;
+
+export const StyledContainer = styled(Container)`
+    && {
+        display: grid;
+        grid-template-rows: 1fr 1fr;
+        justify-content: flex-start;
+        row-gap: 1rem;
+    }
+`;
+
+export const StyledBox = styled(Box)`
+   overflow: hidden;
+`;
+
+export const VerticalLine = styled.div(({ theme }) => `
+    width: 1px;
+    height: 95%;
+    background-color: ${theme.palette.grey[300]};
+    margin-top: 5px;
```

Comment:
i'd prefer to keep height: 95% and instead of margin top center content vertically